docs/install/README.md

```diff
@@ -835,7 +835,7 @@ on Docker Hub for a list of available images.
 Install with kubectl:
 
 ```console
-kubectl apply -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane
+kustomize build github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane\?ref\=v1.0.0 | kubectl apply --server-side --force-conflicts -f -
```

Comment:
Can you update the version to v1.0.2?

Reply:
Are we going to update this each time we push a new release?
Sounds a little painful to me 😕

Reply:
Also, why are we using that method instead of

``kubectl apply --server-side --force-conflicts -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane``

?

Reply:
AFAIK, it was because we're using Kustomize features that aren't available in the version that ships with `kubectl`.

Reply:
> AFAIK, it was because we're using Kustomize features that aren't available in the version that ships with `kubectl`.

This is correct.

Reply:
>Are we going to update this each time we push a new release?
>Sounds a little painful to me

I agree that it is a bit painful, but let me ask you this. How likely do you think it is for someone who is using this install guide to simply copy and paste the code snippets? 😄

Reply:
I agree that's a problem. But I'm pretty sure we won't (and shouldn't) think about updating this each time we cut out a release.
We need to either template this in some way or put notes that the version number should be updated and put a placeholder instead.